darts/timeseries.py

```diff
@@ -3191,6 +3189,18 @@ def plot(
 
             if central_series.shape[0] > 1:
                 p = central_series.plot(*args, **kwargs)
+            # empty TimeSeries
+            elif central_series.shape[0] == 0:
+                p = plt.plot(
+                    [],
+                    [],
+                    *args,
+                    **kwargs,
+                )
+                # make the emptiness of the TimeSerie obvious
+                plt.xlim(-0.2, 5.2)
```

Comment:
Looks good - maybe just check that this doesn't affect plotting other series on the same figure later on.

Reply:
It was indeed interfering with the others plots, I removed this line. Should I also remove the empty TimeSeries from the labels or is it fine if there is a legend of an non-existing/appearing curve in the plot?

Reply:
It's fine I think.